plugins/scaffolder/api-report.md

```diff
@@ -581,6 +582,9 @@ export type ScaffolderTaskStatus = ScaffolderTaskStatus_2;
 // @public @deprecated (undocumented)
 export type ScaffolderUseTemplateSecrets = ScaffolderUseTemplateSecrets_2;
 
+// @public (undocumented)
+export const Secret: (props: ScaffolderRJSFFieldProps) => React_2.JSX.Element;
```

Comment:
i see that other controls also follow this simple pattern, but with this single-word control it looks like it'd been nice if they'd been grouped under some namespace or at least been named `FooFormControl` or whatnot

Reply:
Yeah, I wonder if we can avoid exporting this raw component actually, and just export the field extension itself.